resources/streams/config/currencies.php

```diff
@@ -1245,8 +1245,15 @@
             'separator' => ',',
             'point' => '.',
             'decimals' => 2,
-            'symbol' => '$',]
-
-
+            'symbol' => '$',
+        ],
+        'PER' => [
+            'name' => 'Percentage',
+            'direction' => 'ltr',
+            'separator' => ',',
+            'point' => '.',
+            'decimals' => 4,
+            'symbol' => '%'
+        ]
```

Comment:
why did you add percentage in currencies? percentage is not a currency.

Reply:
It will be useful for users who wants to show percentage fields instead of price.

Reply:
Why is the percentage shown instead of the price? How can you compare the percentage?